packages/fxa-auth-server/test/remote/subscription_tests.js

```diff
@@ -199,8 +200,9 @@ describe('remote subscriptions:', function() {
           {
             subscription_id: subscriptionId,
             plan_id: PLAN_ID,
-            current_period_end: Date.now() / 1000,
-            current_period_start: Date.now() / 1000,
+            created: date,
+            current_period_end: date,
+            current_period_start: date,
```

Comment:
why are these changing from `Date.now() / 1000` to `Date.now() * 1000`?

Reply:
Well, the value for this particular part doesn't really matter -  it just is ensuring that the dates are going to have the same result. I was having occasional issues when debugging with the results of `Date.now()` not being the same, so I changed that bit to be easier.

The `Date.now() / 1000` to `Date.now() * 1000` is actually a result of issues I was having with the `listActive` test, I am addressing those via @6a68 's comments.